src/components/Content/ContentSettings.vue

```diff
@@ -80,6 +80,20 @@
                 for="showVirtualLevels"
               >Show Virtual Levels Beyond {{maxLevel}}</label>
             </div>
+            
+            <div class="custom-control custom-switch">
+              <input
+                v-model="showFullValues"

```

Comment:
You need to set up a computed property to bind your `showFullValues` to down below. It should look something like
```
    showFullValues: {
      get() {
        return this.$store.getters["settings/showFullValues"];
      },
      set(value) {
        this.$store.commit("settings/setShowFullValues", value);
      }
    },
```
Look under the other things under `computed` to get an idea here